app/graphql/types/profile.rb

```diff
@@ -2,14 +2,15 @@ class Types::Profile < Types::BaseObject
   description 'A user profile on Kitsu'
 
   field :id, ID, null: false
+
   field :slug, String,
     null: true,
     description: 'The URL-friendly identifier for this profile'
+
   field :url, String,
     null: true,
-    description: 'The URL for this profile'
+    description: 'The full URL for this profile'
 
-  # Name
   field :name, String,
     null: false,
     description: 'A non-unique, user-visible name for the profile.  Can contain spaces, emoji, etc.'
```

Comment:
A non-unique publicly visible name for the profile. Minimum of 3 characters and any Unicode character is valid.